src/main/java/org/powerbot/script/rt4/Game.java

```diff
@@ -547,6 +551,104 @@ public Component mapComponent() {
 		return new Component(ctx, i, -1);
 	}
 
+	/***
+	 *
+	 * 	Sets the mouse toggle button to return to expected clicking for mobile.
+	 * @return true if it in expected state.
+	 *
+	 * @param enabled Expected state you want for the toggle
+	 * @return true if in expected state.
+	 */
+	private boolean setMouseToggle(boolean enabled) {
```

Comment:
should this take an enum of MouseToggle?

Reply:
The idea would be enable to enable/disable it rather than changing it. Maybe the name could be changed to setMouseToggleEnabled() rather than relying on the value passed.

Reply:
Maybe rename MouseToggle enum to MouseToggleState or something, jsut seems confusing setMouseToggle is a boolean but MouseToggle is an enum

Reply:
MouseToggleAction for enum name then maybe name the method setMouseActionEnabled?